.github/workflows/publish-gh-registry.yml

```diff
@@ -1,6 +1,7 @@
 name: publish to github registry
 env:
   GITHUB_PASSWORD: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Lets remove this

Reply:
This is required to publish to our internal github npm for testing in backstage dev env

Reply:
Oh right.

Reply:
@jdhayford Github says NAH for `GITHUB_` which is why I called it out https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#naming-conventions-for-configuration-variables